### ISSUE TITLE
Background audio not working on Web App

### ISSUE THREAD

**Issue by IronMaltese** [snuffyDev/Beatbump#10]
I’m on iOS 15.1 and the WebApp doesn’t play audio in the background, also keep up the phenomenal work, I’ve been looking for a YouTube music frontend for awhile

**Comment by snuffyDev** [comment]
Hey, my apologies for a late response! 

If you're talking about it being 'installed' with 'add to homescreen', then unfortunately only on iOS does this issue happen. There's been an [open bug report](https://bugs.webkit.org/show_bug.cgi?id=198277) on the WebKit bug tracker for quite some time, so unfortunately as a PWA I can't fix it. I'm also an iOS user and it's honestly annoying. For Beatbump, the way I get around it is to keep it pinned to my favorites and then I just use it like I would normally through Safari.

However, this issue can also happen when being used through Safari normally. Luckily, I was able to figure out a way to fix it through the settings for Safari. [See this short guide](https://github.com/snuffyDev/Beatbump/wiki/Fixing-Background-Autoplay-on-iOS-15) I wrote up.

Try using it directly through Safari after doing what I described in the guide and let me know if it worked!
